config.json

```diff
@@ -43,6 +43,36 @@
     "Firefox ESR"
   ],
   "settings": {
+    "channel_selector": {
+      "channels": [
+        {
+          "id": 1,
+          "name": "Channel_1",
+          "code": "code_1",
+          "is_active": false,
+          "switch_url":"/?setChannelId=1"
+        },
+        {
+          "id": 2,
+          "name": "Channel_2",
+          "code": "code_2",
+          "is_active": true,
+          "switch_url":"/?setChannelId=2"
+        },
+        {
+          "id": 3,
+          "name": "Channel_3",
+          "code": "code_3",
+          "is_active": false,
+          "switch_url":"/?setChannelId=3"
+        }
+      ],
+      "active_channel_id": 2,
+      "active_channel_name": "Channel_2",
+      "active_channel_code": "code_2",
+      "default_channel_code": "code_2"
```

Comment:
Do we really need this data? If we store active, can we just get the rest from array?

Reply:
fixed!